ethportal-api/src/types/content_key.rs

```diff
@@ -15,6 +15,11 @@ pub trait OverlayContentKey:
     fn content_id(&self) -> [u8; 32];
     /// Returns the bytes of the content key.
     fn to_bytes(&self) -> [u8; 33];
+    /// Returns the content key as a hex encoded "0x"-prefixed string.
+    fn to_hex_string(&self) -> String {
```

Comment:
I prefer `to_hex()` here. Imo, it's implicit that a fn named `to_hex()` will return a hex-encoded string. However, that might be a hangover from spending a fair amount of time from working with the [eth-utils python library](https://eth-utils.readthedocs.io/en/stable/utilities.html#to-hex-bytes-int-bool-text-str-hexstr-str-hexstr). I'm not sure if people share this convention?

Reply:
No problem